.github/workflows/release.yaml

```diff
@@ -329,6 +329,26 @@ jobs:
           -d "{\"event_type\": \"auto-kotsadm-update\", \"client_payload\": {\"version\": \"${GIT_TAG:1}\" }}" \
           "https://api.github.com/repos/replicatedhq/kurl/dispatches"
 
+  tag-helm-chart:
+    runs-on: ubuntu-20.04
+    needs: [generate-tag, generate-kurl-addon-pr]
```

Comment:
I added `generate-kurl-addon-pr` as a `needs` as I wanted this to run at the end of the CI, ensuring all other previous steps needed for a valid release complete.